show/vlan.py

```diff
@@ -4,105 +4,133 @@
 
 import utilities_common.cli as clicommon
 
+
 @click.group(cls=clicommon.AliasedGroup)
 def vlan():
     """Show VLAN information"""
     pass
 
+
+def get_vlan_id(ctx, vlan):
+    vlan_prefix, vid = vlan.split('Vlan')
+    return vid
+
+
+def get_vlan_ip_address(ctx, vlan):
+    cfg, _ = ctx
+    _, vlan_ip_data, _ = cfg
+    ip_address = ""
+    for key in vlan_ip_data:
+        if not clicommon.is_ip_prefix_in_key(key):
+            continue
+        ifname, address = key
+        if vlan == ifname:
+            ip_address += "\n{}".format(address)
+
+    return ip_address
+
+
+def get_vlan_ports(ctx, vlan):
+    cfg, db = ctx
+    _, _, vlan_ports_data = cfg
+    vlan_ports = []
+    iface_alias_converter = clicommon.InterfaceAliasConverter(db)
+    # Here natsorting is important in relation to another
+    # column which prints port tagging mode.
+    # If we sort both in the same way using same keys
+    # we will result in right order in both columns.
+    # This should be fixed by cli code autogeneration tool
+    # and we won't need this specific approach with
+    # VlanBrief.COLUMNS anymore.
+    for key in natsorted(list(vlan_ports_data.keys())):
+        ports_key, ports_value = key
+        if vlan != ports_key:
+            continue
+
+        if clicommon.get_interface_naming_mode() == "alias":
+            ports_value = iface_alias_converter.name_to_alias(ports_value)
+
+        vlan_ports.append(ports_value)
+
+    return '\n'.join(vlan_ports)
+
+
+def get_vlan_ports_tagging(ctx, vlan):
+    cfg, db = ctx
+    _, _, vlan_ports_data = cfg
+    vlan_ports_tagging = []
+    # Here natsorting is important in relation to another
+    # column which prints port tagging mode.
```

Comment:
"port tagging" is a copy-paste error from previous function. Should be port list or equivalent